README.md

```diff
@@ -61,6 +61,39 @@ module "vpc" {
 For Terraform version older than 0.7.0 use `ref=v1.0.0`:
 `source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.0"`
 
+Usage with other modules
+------------------------
+
+There are situations where other modules will be used alongside this module, but
+all the networking components out of this module (such as route tables, NAT
+Gateways and Internet Gateways) needs to be provisioned first before resources
+(such as EC2 instances) can be created in other modules. Use of `depends_id` as
+an output variable provides a suitable workaround.
+
+Here is an example of usage alongside the `tf_aws_bastion_s3_keys` community
```

Comment:
I see your point. Thanks for clarifications. Indeed I have not experienced this problem myself. I normally use combination of modules `terraform-community-modules/tf_aws_vpc_only + terraform-community-modules/tf_aws_igw + terraform-community-modules/tf_aws_public_subnet + terraform-community-modules/tf_aws_private_subnet_nat_gateway` instead of `terraform-community-modules/tf_aws_vpc` and that is probably why by the time bastion host instances need IGW it is surely created (just an assumption).

At some cases, I create the whole stack (incl. VPC resources and bastion host) dynamically and have never experienced the problem myself.

I recommend to simplify the code in this PR and leave only `depends_on = ["aws_internet_gateway.mod"]`, because this seems the only dependency which is required (doc is here)[https://www.terraform.io/docs/providers/aws/r/internet_gateway.html].